ssn/rdf/sosa.ttl

```diff
@@ -153,59 +144,52 @@ sosa:Result
+  skos:definition "Feature on which Observations may be made, which is intended to be representative of a FeatureOfInterest that is not fully accessible."@en ;
   skos:example "A 'station' is essentially an identifiable locality where a sensor system or Process may be deployed and an observation made. In the context of the observation model, it connotes the 'world in the vicinity of the station', so the observed properties relate to the physical medium at the station, and not to any physical artifact such as a mooring, buoy, benchmark, monument, well, etc."@en ;
   skos:example "A statistical sample is often designed to be characteristic of an entire population, so that observations can be made regarding the sample that provide a good estimate of the properties of the population."@en ;
   skos:note "A transient sample, such as a ships-track or flight-line, might be identified and described, but is unlikely to be revisited exactly."@en ;
 .
 sosa:Sampler
   rdf:type rdfs:Class ;
   rdf:type owl:Class ;
-  rdfs:label "Sampling device or entity"@en ;
-  skos:definition "A device that is used by, or implements, a sampling Procedure to create or transform one or more samples."@en ;
-  skos:example "Ball mill"@en ;
-  skos:example "Diamond drill"@en ;
-  skos:example "Hammer"@en ;
-  skos:example "Hypodermic syringe and needle"@en ;
-  skos:example "Image sensor"@en ;
-  skos:example "Seive stack"@en ;
-  skos:example "Soil auger"@en ;
+  rdfs:label "Sampler"@en ;
+  skos:definition "A device that is used by, or implements, a Sampling Procedure to create or transform one or more samples."@en ;
+  skos:example "A ball mill, diamond drill, hammer, hypodermic syringe and needle, image sensor or a soil auger can all act as sampling devices (i.e. be Samplers)."@en ;
 .
 sosa:Sampling
   rdf:type rdfs:Class ;
   rdf:type owl:Class ;
-  rdfs:label "Act of Sampling"@en ;
+  rdfs:label "Sampling"@en ;
   skos:definition "An act of Sampling carries out a sampling Procedure to create or transform one or more samples."@en ;
   skos:example "Crushing a rock sample in a ball mill."@en ;
   skos:example "Digging a pit through a soil sequence."@en ;
-  skos:example "Dividing a field site into quadrats."@en ;
+  skos:example "Dividing a field site into quadrants."@en ;
   skos:example "Drawing blood from a patient."@en ;
   skos:example "Drilling an observation well."@en ;
   skos:example "Establishing a station for environmental monitoring."@en ;
   skos:example "Registering an image of the landscape."@en ;
-  skos:example "Seiving a powder to separate the subset finer than 100-mesh."@en ;
+  skos:example "Sieving a powder to separate the subset finer than 100-mesh."@en ;
   skos:example "Selecting a subset of a population."@en ;
   skos:example "Splitting a piece of drill-core to create two new samples."@en ;
   skos:example "Taking a diamond-drill core from a rock outcrop."@en ;
```

Comment:
I would suggest to do the examples in the same style as you did for 'Sampler', namely as text for one skos:example triple.